repack_st_arch.csh

```diff
@@ -472,9 +472,18 @@ if ($do_restarts == true) then
          endif
          touch mycmdfile
          set i = 1
+# When the first day of the month is a Monday (or whatever's set in setup_advance),
+# st_archive archives those files to rest without compressing them.
+# There could be a compression here as part of tarring them:
+#          set comp_arg = ''
+#          if ($day_o_month == 01) comp_arg = ' -z'
+#          ...
+#          echo " tar ${comp_arg} -c ...
+# This would still result in tar files with a different size 
+# for the first day of the month, but would save space (~67 Gb /month).
          while ($i <= $data_NINST)
             set INST = `printf %04d $i`
-            echo "tar -c -f ${yr_mo}/${data_CASE}.${INST}.alltypes.${rd}.tar "                     \
+            echo "tar -c -f ${yr_mo}/${data_CASE}.${INST}.alltypes.${rd}.tar "                 \
```

Comment:
Remind me to discuss the advantages/disadvantages of   'tar -czvf' vs compressing the contents.

Reply:
I've made a note for Monday.